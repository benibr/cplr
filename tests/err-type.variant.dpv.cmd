../cplr '-n' 'puts(1)' -dpv
