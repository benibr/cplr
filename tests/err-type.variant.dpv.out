Generation phase
Generating code
Generating section main
================================================================================
     1	/* main */
     2	#line 169 "internal"
     3	int main(int argc, char **argv) {
     4	    int ret = 0;
     5	/* main */
     6	#line 1 "main_0"
     7	    puts(1);
     8	/* done */
     9	#line 184 "internal"
    10	    return ret;
    11	}
================================================================================
Generated bytes: 174 code, 174 dump
Compiler condition:
main_0:1: warning: implicit declaration of function 'puts'
Compilation succeeded.
Execution succeeded.
